ui/base/src/main/res/values/strings_base_ui.xml

```diff
@@ -5,6 +5,7 @@
 
     <!-- Tool Categories -->
     <eat-comment />
+    <string name="tool_category_label">Categories</string>
```

Comment:
this is a bad name for this string because if in the future we want to introduce a category named "label" this will cause a conflict.

Also, this string should be in `strings_tools.xml` within the `app` module

Reply:
the category names in this section are very specific because of the programmatic lookup that occurs based on values coming back from the API